js/okcoinusd.js

```diff
@@ -272,6 +274,41 @@ module.exports = class okcoinusd extends Exchange {
         let ticker = this.extend (response['ticker'], { 'timestamp': timestamp });
         return this.parseTicker (ticker, market);
     }
+    
+    parseTickers (tickers, timestamp, symbols) {
+        if (!tickers || (tickers.length < 1)) {
```

Comment:
Can't use tickes.length here – it can only wok in a portable way, if tickers is a string.
To get the length of the array in a portable way you should create a variable and use the following syntax:
```JavaScipt
const tickersLength = tickers.length;
if (!tickers || (tickersLength < 1)) {
    ...
```